debian/control

```diff
@@ -12,6 +12,7 @@ Build-Depends:
   libxkbcommon-dev,
   libwayland-dev,
   pkg-config,
+  pop-launcher,
```

Comment:
Should this be a build depends?

Reply:
I feel like it could just be a depends instead